src/conversion/JavaToTypeScript.ts

```diff
@@ -155,6 +157,14 @@ export interface IConverterConfiguration {
 export class JavaToTypescriptConverter {
     public constructor(private configuration: IConverterConfiguration) {
         PackageSourceManager.configure(configuration.javaLib, configuration.options.importResolver);
+
+        configuration.packageRoot = path.resolve(configuration.packageRoot);
+
+        const prefix = configuration.options.prefix;
+        configuration.options.prefix =
+            typeof prefix === 'function'
+                ? prefix
+                : () => prefix as string ?? '';
```

Comment:
Since there are only 2 possible types for `prefix` no cast to `string` is necessary here. I'm pretty sure ESLint will remove the cast. Do you use ESLint? It would also complain about missing braces (that's how it is configure in this repo). Should be: `() => { return prefix ?? ""; };`.

Reply:
I use eslint, of course. But I use intellij, and intellij has a problem with eslint 8.28. So I set it to use 8.22 for my development environment.
All of your comments have been applied. Please check. :)